model:
# type: line2d
# a: 5.
# b: 2.
# type: curve2d_2order
# a: -3.
# b: -2.
# c: 5.
  type: curve2d_3order
  a: 1.
  b: -8.
  c: -5.
  d: -7.
# type: michaelis_menten
# b0: 0.3
# b1: 0.5
# type: cos
# a: 2.
# b: 2.


x_range: [0., 10.]
